libs/image-editor/build.gradle

```diff
@@ -45,40 +45,44 @@ repositories {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation(libs.androidxCore)
+    implementation libs.androidxActivity
+    implementation libs.androidxActivityKtx
+    implementation libs.androidxFragment
+    implementation libs.androidxAppcompat
+    implementation libs.androidxRecyclerview
 
-    implementation "androidx.core:core:$androidxCoreVersion"
-    implementation "androidx.activity:activity:$androidxActivityVersion"
-    implementation "androidx.activity:activity-ktx:$androidxActivityVersion"
-    implementation "androidx.fragment:fragment:$androidxFragmentVersion"
-    implementation "androidx.appcompat:appcompat:$androidxAppcompatVersion"
-    implementation "androidx.recyclerview:recyclerview:$androidxRecyclerviewVersion"
-    implementation "androidx.constraintlayout:constraintlayout:$androidxConstraintlayoutVersion"
-    implementation "androidx.viewpager2:viewpager2:$androidxViewpager2Version"
-    implementation "com.google.android.material:material:$googleMaterialVersion"
+
+    implementation libs.androidxConstraintLayout
+    implementation libs.androidxViewpager2
+    implementation libs.googleMaterial
     implementation "androidx.navigation:navigation-fragment:$gradle.ext.navigationVersion"
```

Comment:
`$gradle.ext.navigationVersion"`  how can we convert this into version catlog?

Reply:
I guess I solved it, thus closing it.  gradle ext version are managed in settings.gradle

```
pluginManagement {
    gradle.ext.kotlinVersion = '1.9.22'
    gradle.ext.kspVersion = '1.9.22-1.0.17'
    gradle.ext.agpVersion = '8.1.0'
    gradle.ext.googleServicesVersion = '4.3.15'
    gradle.ext.navigationVersion = '2.7.7'
    gradle.ext.sentryVersion = '4.3.1'
    gradle.ext.daggerVersion = "2.50"
    gradle.ext.detektVersion = '1.23.0'
    gradle.ext.violationCommentsVersion = '1.70.0'
    gradle.ext.measureBuildsVersion = '2.1.2'
    gradle.ext.koverVersion = '0.7.5'
    gradle.ext.dependencyAnalysisVersion = '1.28.0'
    
```